src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -80,7 +81,13 @@ private Command SetupAnalyzeTemplateCommand()
 
                     foreach (var evaluation in evaluations)
                     {
-                        Console.WriteLine($"{evaluation.RuleName}: {evaluation.RuleDescription}, Result: {evaluation.Passed.ToString()}");
+                        string resultString = GenerateResultString(evaluation);
+                        string fileMetadata = $"\n\tFile: {templateFilePath}";
```

Comment:
template and parameters will be the same for every evaluation.  Can that be printed just once?

Reply:
I meant only print the template and parameters file names once for everything.  Every evaluation here is for the same template, so there's no need to keep reprinting it.

Reply:
I moved it and now it should only be printing once.